src/strands_tools/bright_data.py

```diff
@@ -0,0 +1,486 @@
+    search queries, and extracting structured data from various websites.
+
+    Args:
+    action: The action to perform (scrape_as_markdown, get_screenshot, search_engine, web_data_feed)
+    url: URL to scrape or extract data from (for scrape_as_markdown, get_screenshot, web_data_feed)
+    output_path: Path to save the screenshot (for get_screenshot)
+    zone: Override default Bright Data zone (optional)
+    query: Search query (for search_engine)
+    engine: Search engine to use (google, bing, yandex, default: google)
+    language: Two-letter language code for search results (hl parameter for Google)
+    country_code: Two-letter country code for search results (gl parameter for Google)
+    search_type: Type of search (images, shopping, news, etc.)
+    start: Results pagination offset (0=first page, 10=second page)
+    num_results: Number of results to return (default: 10)
+    location: Location for search results (uule parameter)
+    device: Device type (mobile, ios, android, ipad, android_tablet)
+    return_json: Return parsed JSON instead of HTML/Markdown (default: False)
+    source_type: Type of data source for web_data_feed (e.g., 'linkedin_person_profile', 'amazon_product')
+    num_of_reviews: Number of reviews to retrieve (only for facebook_company_reviews)
+    timeout: Maximum time in seconds to wait for data retrieval (default: 600)
+    polling_interval: Time in seconds between polling attempts (default: 1)
+
+    Returns:
+        str: Response content from the requested operation
+    """
+    try:
+        if not action:
+            raise ValueError("action parameter is required")
+
+        client = BrightDataClient(verbose=True)
+
+        if action == "scrape_as_markdown":
+            if not url:
+                raise ValueError("url is required for scrape_as_markdown action")
+            return client.scrape_as_markdown(url, zone)
+
+        elif action == "get_screenshot":
+            if not url:
+                raise ValueError("url is required for get_screenshot action")
+            if not output_path:
+                raise ValueError("output_path is required for get_screenshot action")
+            output_path_result = client.get_screenshot(url, output_path, zone)
+            return f"Screenshot saved to {output_path_result}"
+
+        elif action == "search_engine":
+            if not query:
+                raise ValueError("query is required for search_engine action")
+            return client.search_engine(
+                query=query,
+                engine=engine,
+                zone=zone,
+                language=language,
+                country_code=country_code,
+                search_type=search_type,
+                start=start,
+                num_results=num_results,
+                location=location,
+                device=device,
+                return_json=return_json,
+            )
+
```

Comment:
To support get_active_zones:

```suggestion

   elif action == "get_active_zones":
            zones = client.get_active_zones()
            return json.dumps(zones, indent=2)

```

Reply:
- Indentation might be off - suggested over GitHub review -